src/main/java/com/app/service/file/parameters/VoltageSweep.java

```diff
@@ -10,51 +10,29 @@
     static final double minSpot = -40;
     static final double maxSpot = 40;
 
-    private double start;
-    private double stop;
-    private double step;
-    private double spot;
-
-
-
-    public double getStart() {
-        return start;
-    }
-
-    public double getStop() {
-        return stop;
-    }
-
-    public double getStep() {
-        return step;
-    }
-
-    public double getSpot() {
-        return spot;
-    }
 
     public void setStart(Double start) {
         if(start < minStart || start > maxStart){
-            this.start = minStart;
-        }else this.start = start;
+            super.start = minStart;
```

Comment:
nebolo by rozumnejsie nastavit hodnotu na max ak je vacsia ako max a na min, ak je menas ako min? myslim, ze to je najlogickejsie, co na to povies?